ui-svelte-starter/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "com.adobe.uxp.starter.svelte",
+  "version": "1.0.0",
+  "scripts": {
+    "watch": "webpack -w --mode production",
```

Comment:
Thought I'd point out that I realized I could run webpack in watch mode, as opposed nodemon to keep the dependency count low. It was working fine for me during development, but let me know if I should change this to nodemon for consistency or for any other technical reason. I'm happy to change it!